tests/test_middlewares.py

```diff
@@ -55,9 +55,17 @@ def process_request(self, req, resp):
         req.path = req.path.replace('/base_path', '', 1)
 
 
+class AccessParamsMiddleware(object):
+
+    def process_resource(self, req, resp, resource, params):
+        global context
+        params['added'] = True
```

Comment:
Would you mind adding a test to check that the modified parameter is getting propagated correctly to the responder method on the resource?